integration/mesos/pom.xml

```diff
@@ -39,7 +39,7 @@
     </dependency>
     <dependency>
       <groupId>com.google.protobuf</groupId>
-      <artifactId>protobuf-java</artifactId>
+       <artifactId>protobuf-java</artifactId>
```

Comment:
also, if we build alluxio with -Pmesos, we end up including this dependency in the assembly uber jar ... is that okay?